draft-ietf-quic-transport.md

```diff
@@ -833,11 +833,6 @@ implementations.  As an optimization, an endpoint could send frames related to
 flow control only when there are other frames to send or when a peer is blocked,
 ensuring that flow control does not cause extra packets to be sent.
 
-When a sender receives credit after being blocked, it might send a large amount
-of data in response. As is recommended in {{QUIC-RECOVERY}}, implementations
-should pace this data to avoid sending it in a burst and causing short-term
```

Comment:
I think this cross-reference is needed - people implementing flow control need to ensure this part is correctly implemented also.

Reply:
I've added a reference in but with better text I think.